install.R

```diff
@@ -0,0 +1,7 @@
+install.packages("tidyverse")
+install.packages("effsize")
+install.packages("prmisc")
+install.packages("binb")
```

Comment:
This line can be deleted, right? Because `prmisc` is installed from GitHub later and this line will fail anyways?

Reply:
Ah you mean to delete the line
```diff
-install.packages("prmisc")
```
Yes, sounds reasonable!